spec/webauthn_spec.rb

```diff
@@ -43,8 +43,8 @@
     end
 
     context "Relying Party info" do
-      it "has relying party name default" do
-        expect(@credential_creation_options[:rp][:name]).to eq("web-server")
+      it "has relying party name default set to nothing" do
+        expect(@credential_creation_options[:rp][:name]).to eq(nil)
```

Comment:
Just realized, if released, this would be a breaking change. The intention was to deprecate `WebAuthn.credential_creation_options` without breaking compatibility.

Needs to be fixed/restored.

Reply:
https://github.com/cedarcode/webauthn-ruby/pull/265